app/controllers/find/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Find::ApplicationController < ActionController::Base
```

Comment:
Can we use the module https://rubystyle.guide/#namespace-definition

Reply:
Done https://github.com/DFE-Digital/publish-teacher-training/pull/2922/commits/4914f94290b8a9bc68665fc0c7b7c5ce1b301898